packages/fixie/src/auth.ts

```diff
@@ -0,0 +1,207 @@
+import yaml from 'js-yaml';
+import fs from 'fs';
+import terminal from 'terminal-kit';
+import os from 'os';
+import path from 'path';
+import untildify from 'untildify';
+import axios from 'axios';
+import open from 'open';
+import http from 'http';
+import crypto from 'crypto';
+import readline from 'readline';
+import net from 'net';
+
+const { terminal: term } = terminal;
+
+import { FixieClient } from './client.js';
+
+/** Represents contents of the Fixie CLI config file. */
+export interface FixieConfig {
+  apiUrl?: string;
+  apiKey?: string;
+}
+
+export const FIXIE_API_URL = 'https://api.fixie.ai';
+export const FIXIE_CONFIG_FILE = '~/.config/fixie/config.yaml';
```

Comment:
Plz use `os.homedir()` and `path.join` for cross-platform compat https://nodejs.org/api/os.html#oshomedir.

Reply:
untildify handles this for us!